llvm/include/llvm/IR/IntrinsicInst.h

```diff
@@ -591,6 +592,31 @@ class VPCmpIntrinsic : public VPIntrinsic {
   /// @}
 };
 
+/// This is the common base class for floating point builtin intrinsics.
+class FPBuiltinIntrinsic : public IntrinsicInst {
+public:
+  static const std::string FPBUILTIN_PREFIX;
+  static const std::string FP_MAX_ERROR;
```

Comment:
Nit: Should we call ths FPBUILTIN_MAX_ERROR just to be in sync with attribute name?